config/sqs/config.go

```diff
@@ -16,6 +16,13 @@ func Configure(p *config.Provider) {
 	})
 
 	p.AddResourceConfigurator("aws_sqs_queue", func(r *config.Resource) {
+		r.Sensitive.AdditionalConnectionDetailsFn = func(attr map[string]any) (map[string][]byte, error) {
+			conn := map[string][]byte{}
+			if a, ok := attr["id"].(string); ok {
+				conn["queue_url"] = []byte(a)
```

Comment:
```suggestion
			if a, ok := attr["url"].(string); ok {
				conn["url"] = []byte(a)
```

The above configuration is better in order to adapt to the terraform document since the requested parameter is the url in the relevant issue. What do you think?

Reply:
I agree.